spec/modules.md

```diff
@@ -52,9 +52,13 @@ An *empty CSS tree* contains no statements.
 ### Configuration
 
 A *configuration* is a map from [variable](variables.md) names to SassScript
-values. An *empty configuration* contains no entries.
+values. An *empty configuration* contains no entries. Each *configuration* has
+a uniquely-identifying opaque ID.
```

Comment:
Nit: I'd include the ID in the first sentence, since it's (now) part of the structure of a configuration:

```suggestion
values and an opaque ID. An *empty configuration* contains no entries.
```

I'd omit "uniquely-identifying" because this is so far just describing the _structure_ of a configuration, not its _semantics_.